pages_builder/generate_pages.py

```diff
@@ -87,7 +94,41 @@ def render_page(self, input_file):
                 partial['pageTitle'] +
                 " - Digital Marketplace frontend toolkit"
             )
-            if "examples" in partial:
+            if (
```

Comment:
Could we remove the brackets from this and line 105 to match the style adopted for the rest of this file?